CONTRIBUTING.md

```diff
@@ -219,6 +219,45 @@ name.
 </record>
 ```
 
+### External dependencies
+
+#### `__openerp__.py`
+If your module use extras dependencies of python or binaries you should add to `__openerp__py` file the section `external_dependencies`.
+
+```python
+{
+    'name': 'Module of example',
+    'depends': ['base'],
+    ...
+    'external_dependencies': {
+        'bin': [
+            'external_dependency_binary_1',
+            'external_dependency_binary_2',
+            ...
+            'external_dependency_binary_N',
+        ],
+        'python': [
+            'external_dependency_python_1',
+            'external_dependency_python_2',
+            ...
+            'external_dependency_python_N',
+        ],
+    },
+    ...
+    'installable': True,
+}
+```
+
+#### ImportError
+In python files where you use a `import external_dependency_python_N` you will need to add a `try-except` with a info log.
+
+```python
+try:
```

Comment:
I like our standard of use import sorted in top file of python:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

If you need use your library in many method you will need to add a `import`.

I undertand your point:
`you don't receive any annoying log line with uninstalled modules`
I have a proposal for you: https://github.com/OCA/maintainer-tools/pull/92/files#r35818714